src/tokens.ts

```diff
@@ -158,6 +160,14 @@ export const TOKEN_SYMBOLS_MAP = {
       [CHAIN_IDs.POLYGON]: "0x3066818837c5e6eD6601bd5a91B0762877A6B731"
     },
   },
+  USDB: {
+    name: "USDB",
+    symbol: "USDB",
+    decimals: 18,
+    addresses: {
+      [CHAIN_IDs.BLAST]: "0x4300000000000000000000000000000000000003",
```

Comment:
Should we add testnet here? [USDB](https://sepolia.blastexplorer.io/address/0x4200000000000000000000000000000000000022)

Reply:
I included it initially, but I don't have a good reference for DAI on Sepolia, so there's currently nothing on Sepolia to map it back to. If we can find a "canonical" DAI to support on Sepolia then I'm on board with adding it; otherwise we should be OK with just WETH I think.